core/uri.c

```diff
@@ -335,7 +335,7 @@ int uri_toString(const lwm2m_uri_t * uriP,
             }
         }
     }
-    else if(uriP)
+    else if(NULL != uriP)
```

Comment:
Is there a reason to write `NULL != uriP` instead of `uriP != NULL` ?
I feel the second a bit more "natural" ?

Reply:
Personally I prefer the readability of (variable == NULL) but (NULL == variable) is often considered safer since if you mistype "==" as "=" the latter causes compilation error.
However "follow the existing coding style" usually is considered most important (I see plenty of both). Shall I change? (need to solve "eca" problem with husqvarna e-mail anyway)

Reply:
@qleisan thx for the explanation.

I think both solution makes sense.
I guess the team should choose the way which should be applied for this project.
This makes me thing that this could be the beginning of a kind of code styling & rules for this project : #513.

Reply:
Regarding the safety: I think this is no longer relevant as we have compiler warnings for this nowadays.